packages/HorizonUI/Sources/HorizonUI/Sources/Components/Button/HorizonUI.ButtonStyles.swift

```diff
@@ -42,9 +53,40 @@ extension HorizonUI {
             self.fillsWidth = fillsWidth
             self.leading = leading
             self.trailing = trailing
+
+            self.badge = nil
+            self.badgeStyle = nil
+            self.icon = nil
+        }
+
+        fileprivate init(
+            background: any ShapeStyle,
+            foreground: Color,
+            badgeStyle: HorizonUI.Badge.Style,
```

Comment:
Here please rename it to foregroundColor & please rename background to backgroundColor